### PR TITLE
Feature/improve batch messaging

### PR DESCRIPTION
**Pull request by siemensikkema**
Implements #29

### CODE REVIEW

File: Sources/FCM/Helpers/FCM+SendMessage.swift [hunk 1]
```diff
@@ -34,7 +34,6 @@ extension FCM {
 
             return self.client.post(URI(string: url), headers: headers) { (req) in
                 struct Payload: Content {
-                    let validate_only: Bool = false
```
Comment (MihaelIsaev) [inline Sources/FCM/Helpers/FCM+SendMessage.swift:37]: Isn't it required?

**Review by MihaelIsaev — COMMENTED** [review]
@siemensikkema overall new batch send works well? I just can't test it at the moment.

File: Sources/FCM/Helpers/FCM+SendMessage.swift (hunk 1 above)
Comment (siemensikkema) [reply Sources/FCM/Helpers/FCM+SendMessage.swift:37]: Didn't seem to be for me when I tested it. I'll have a look at the documentation.

**Comment by siemensikkema** [comment]
We'll test it more on a project tomorrow. So far it seemed to work well for single notifications sent using the batch method.

File: Sources/FCM/Helpers/FCM+SendMessage.swift (hunk 1 above)
Comment (siemensikkema) [reply Sources/FCM/Helpers/FCM+SendMessage.swift:37]: It doesn't explicitly say it [here](https://firebase.google.com/docs/reference/fcm/rest/v1/projects.messages/send) that is optional but the description does somewhat imply it:

> Flag for testing the request without actually delivering the message.

It seems to work fine without the property in my tests. Removing it was the easiest way to get rid of an Xcode warning that was related to the strict being `Decodable` (due to `Content`) while the property has a default value.

**Comment by siemensikkema** [comment]
> Are we sure the limit is 500?

Pretty sure. [The documentation](https://firebase.google.com/docs/cloud-messaging/send-message#send-a-batch-of-messages) mentions a maximum of 500 and [this python library](https://github.com/firebase/firebase-admin-python/blob/ba12bda0d550c33a2831b02ab31a47b32d06aad3/firebase_admin/messaging.py#L362) also has a limit of 500.

**Comment by pbodsk** [comment]
But [earlier in the same document](https://firebase.google.com/docs/cloud-messaging/send-message#send-messages-to-multiple-devices) they mention

> You can specify up to 100 device registration tokens (500 for Java and Node.js) per invocation.

But that must be legacy since they also support 500 messages in Python but only mentions Java and Node.js.

**Comment by MihaelIsaev** [comment]
> We'll test it more on a project tomorrow. So far it seemed to work well for single notifications sent using the batch method.

@siemensikkema could you please confirm that you tested it for multiple notifications sent using the batch method and I'll merge and release it. Thank you 🙏

**Review by MihaelIsaev — APPROVED** [review]

**Comment by MihaelIsaev** [comment]
@siemensikkema have you had a chance to test it on sending a message to more than one recipient? 🙂

**Comment by siemensikkema** [comment]
@MihaelIsaev Hi! Thanks for following up! Yes, we've been using this in production for over a week now and there are no complaints from the client anymore ☺️

**Comment by MihaelIsaev** [comment]
It is now available since `2.9.0` tag!
@siemensikkema many thanks for the contribution! 🥳🚀

**Pull request merged**